pombola/core/views.py

```diff
@@ -508,11 +509,113 @@ def position(request, pt_slug, ok_slug=None, o_slug=None):
         context['alphabetical_link_from_query_parameter'] = True
     context['positions'] = positions
 
-    return render_to_response(
-        template,
-        context,
-        context_instance=RequestContext(request)
-    )
+    if request.GET.get('view') == 'csv':
+
+        response = HttpResponse(content_type='text/csv')
```

Comment:
With `text/csv` my browser downloads the file, rather than just displaying it. So while this is technically more _correct_, I wonder if it might be more convenient if the content type was `text/plain`, making it easier to check in the browser?

Reply:
Given that this is mostly going to feed things behind the scenes, I think the correct format is probably more useful (and may well stop browsers trying to do interesting things with format extensions such as appending a `.txt` on download, which is definitely wrong.

Reply:
@jacksonj04 Yep, makes sense.